api/v1alpha1/tempooperatorconfig_types.go

```diff
@@ -0,0 +1,39 @@
+package v1alpha1
```

Comment:
I would prefer to have this in a different package e.g. /api/config/...

Reply:
mmh not sure, this is a Kind, as the other one, wondering the tools for generate manifests etc.. will break with this change.

Reply:
we might need to change the api path, but it should be possible to put it in any directory.

Reply:
it should be possible, but it will break a little bit the directory structure, `api/<version>/ ` also, this is part of the API, is another CRD that belongs to this operator.

Reply:
It is part of the operator API, not the tempo API, that is the motivation to move it to a separate package to isolate them.

Reply:
Operator-sdk has a option to define a group. Iam not 100% sure, but would it be a different group?

```
./bin/operator-sdk-v??? create api --group config --version v1alpha1 --kind OperatorConfig
```